docs/secure-connections/sauce-connect-5/installation/docker.md

```diff
@@ -33,8 +33,27 @@ Here are some benefits/use cases for using containerized Sauce Connect Proxy:
     </details>
 2. To run the Sauce Connect Proxy Docker image, modify and run the script below.
 
+   ```shell
+   #!/bin/bash
+
+   # This command sets your environment variables in your current shell. 
+   # Replace the placeholders with your username and access key.
+   SAUCE_USERNAME="<enter username here>" SAUCE_ACCESS_KEY="<enter access key here>" SAUCE_REGION="us-west" SAUCE_TUNNEL_NAME="my-tunnel-name" \
+
+   # This command runs the docker container, integrating the newly set environment variables
+   docker run \
+       -e SAUCE_USERNAME=${SAUCE_USERNAME} \
+       -e SAUCE_ACCESS_KEY=${SAUCE_ACCESS_KEY} \
+       -e SAUCE_REGION=${SAUCE_REGION} \
+       -e SAUCE_TUNNEL_NAME=${SAUCE_TUNNEL_NAME} \
+       --network="host" \
+       -it saucelabs/sauce-connect:5.0
```

Comment:
This could go to the top line. Also using latest or not using 5.0 would make it more future proof.

Reply:
I don't believe you can, I am getting an error trying to run the command with these 2 lines at the top. Perhaps the env vars needs to be specified ahead of the rest.